proto/src/lib.rs

```diff
@@ -13,6 +13,7 @@
 
 #[allow(dead_code)]
 #[allow(clippy::all)]
+#[allow(bare_trait_objects)]
```

Comment:
Can you fix it in the grpcio-compiler?

Reply:
I found related code generation is here: https://github.com/tikv/protobuf-build
This fix work may require the help of @nrc

Reply:
@BusyJay I found related code generation is here: https://github.com/tikv/protobuf-build, not in codegen.rs in grpc-compiler.
So this fix work may require the help of @nrc

Reply:
Have you tried updating the protobuf-build dependency and running a re-generation?

Reply:
> Have you tried updating the protobuf-build dependency and running a re-generation?

Thanks for suggestion, I will try tomorrow.